app/assets/stylesheets/application/modules/_exercise_assignment.scss

```diff
@@ -6,7 +6,28 @@
   margin: 10px 0;
 }
 
-.editor-overlap {
+.mu-editor-overlap {
   z-index: 0;
   position: relative;
 }
+
+.mu-editor {
+  .form-group {
+    margin: 10px 0;
+  }
+  input[type=radio] {
```

Comment:
@flbulgarelli shall we add more styles to the checkboxes and radio buttons? like complementary color for the check?